code/game/objects/items/weapons/traps.dm

```diff
@@ -132,9 +132,18 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
 
 //Using a crowbar allows you to lever the trap open, better success rate
 /obj/item/weapon/beartrap/attackby(obj/item/C, mob/living/user)
```

Comment:
Non-blocker -- would it be possible to add code to trigger traps by throwing things at them?